CHANGELOG.md

```diff
@@ -8,7 +8,14 @@
 
 ### Improvements
 
+> ![Important]
+> This version enables the experimental view renderer used by Session Replay by default.
```

Comment:
`h`: Rolling out a renderer called experimental sounds a bit weird to me. Maybe we rename it to V2 or something.

Reply:
It's still an experimental view renderer just enabled by default. Why should this be renamed if it is still in experimental state?

Reply:
It feels weird to enable something experimental per default for a GA product. If we're confident enough to enable it per default for everyone it shouldn't be experimental anymore IMO.